src/Chapter1/SumOfFirstNMultiples.hs

```diff
@@ -0,0 +1,19 @@
+module Chapter1.SumOfFirstNMultiple where
+sumOfNmultiples :: Int -> Int -> Int -> Int -> Int
```

Comment:
you have two iterators in this solution -- `i` and `count`? Can you modify the solution to use a single iterator?